### PR TITLE
DatabasePager implementation

### PR DESCRIPTION
**Pull request by jtorresfabra**

### CODE REVIEW

**Comment by cedricpinson** [comment]
Is it ready to be reviewed ?

**Comment by jtorresfabra** [comment]
Now it is :)

**Comment by cedricpinson** [comment]
sorry you would need to rebase your PR

**Comment by jtorresfabra** [comment]
Done

File: sources/osgDB/DatabasePager.js [hunk 1]
```diff
@@ -0,0 +1,367 @@
+/**
+ * @author Jordi Torres
+ */
+
+
+define( [
+    'q',
```
Comment (cedricpinson) [inline sources/osgDB/DatabasePager.js]: use q in lower case because the file is supposed to be like that, but you should use Q in the code

**Pull request merged**